ToDo-GUI/README.md

```diff
@@ -0,0 +1,37 @@
+# To-Do App
+
+
+## Introduction
+This project is about creating a ToDo App with simple to use GUI, where the users can add items to their list, remove items and save the list.
+
+
+
+## Getting started
+In order to run this script, you need to have Python and pip installed on your system. After you're done installing Python and pip, run the following command from your terminal to install the requirements from the same folder (directory) of the project.
+```
+pip install -r requirements.txt
+```
+
+After satisfying all the requirements for the project, Open the terminal in the project folder and run
+```
+python todo.py
+```
+or
+```
+python3 todo.py
+```
+depending upon the python version. Make sure that you are running the command from the same virtual environment in which the required modules are installed.
+
+
+![Demo pic of ToDo App by akr](image/akr.jpg)
+
+Now you are all set to explore the ToDo App. Happy Hacking!!!!!!
+
+
+## Author
+[a-k-r-a-k-r](https://github.com/a-k-r-a-k-r)
+
+
+## Reporting Bugs
+Feel free to report any buys or issues at the below github profile.
```

Comment:
After being merged this will be a part of this project, so all the bugs will be reported through raising an issue. Please update the info as such.